sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeCreator.scala

```diff
@@ -403,10 +403,11 @@ case class CreateNamedStruct(children: Seq[Expression]) extends Expression {
        {"a":"1","b":"2","c":"3"}
       > SELECT _FUNC_('a');
        {"a":null}
-  """)
+  """,
+  since = "2.0.1")
```

Comment:
When I go to confirm this `since` tag, I looked at the original PR #13990.

The author implemented codegen version actually, but did remove it based on the comment: https://github.com/apache/spark/pull/13990#discussion_r69660241

I think the comment makes sense to me. To add or not codegen looks not a big deal. This implementation looks simple, anyway.

I have no special option to add it or not. cc @cloud-fan

Reply:
Ah, I see.